src/agent/application.cpp

```diff
@@ -81,7 +81,7 @@ void Application::Init(void)
     mRestWebServer.Init();
 #endif
 #if OTBR_ENABLE_DBUS_SERVER
-    mDBusAgent.Init();
+    SuccessOrDie(mDBusAgent.Init(), "Failed to initialize DBus Agent");
```

Comment:
We can make `Init()` return void and SuccessOrDie in Init itself?

Reply:
Done.